lib/pure/collections/sets.nim

```diff
@@ -293,8 +293,6 @@ proc excl*[A](s: var HashSet[A], key: A) =
   var i = rawGet(s, key, hc)
   var msk = high(s.data)
   if i >= 0:
-    s.data[i].hcode = 0
```

Comment:
Why was that removed?

Reply:
Those assignments are redundant since the same assignments will be done in the first loop iteration anyways. I didn't include them for the new OrderedSet.excl, so I removed them from HashSet.excl for consistency.